### DISCUSSION TITLE
Common `boundary` option for all modifiers?

### DISCUSSION THREAD

**Discussion by rohit2sharma95** [floating-ui/floating-ui#1277] [Q&A] [answered]
Is it not possible to give the value for the `boundary` option for a single instance and its all modifiers at once?

If we pass `document.body` as a value for the `boundary` option in `preventOverflow` modifier, the instance still has `'clippingParents'` value for `boundary` option in `flip` modifier.

**Comment by FezVrasta** [comment, marked as answer]
Hi, no it's not possible, usually one doesn't want to use the same boundary for different modifiers, they all have different requirements.
